obs-script/obs-Auto-League.py

```diff
@@ -242,20 +242,21 @@ def league_data():
     division = league_data['division']
     show_division(division)
     config = [league_data['blue_config_path'], league_data['orange_config_path']]
+    set_logo(config[0], config[1])
     return config
 
 def show_division(div_num):
     # TODO: add divisions
     pass
 
-def get_logo(blue_config, orange_config): #reused for logo later
+def set_logo(blue_config, orange_config): #reused for logo later
     blue_logo = None
     orange_logo = None
     with open(blue_config) as blue_config_file:  # Use file to refer to the file object
         data = blue_config_file.read()
         data = data.split('\n')
         for line in data:
-            del_part = len('participant_config_0 = ')
+            del_part = len('logo = ')
             blue_logo = line[del_part:]
```

Comment:
Is it feasible to import the BotConfigBundle class from rlbot and use that? https://github.com/RLBot/RLBot/blob/master/src/main/python/rlbot/parsing/bot_config_bundle.py#L27-L33

The logo finding logic already exists there, and if you reference it you'll be protected from any changes.

Reply:
Yes, that should definitely be possible.